Neos.ContentRepository.Core/Classes/Factory/CommandHooksFactoryDependencies.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Neos.ContentRepository package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\ContentRepository\Core\Factory;
+
+use Neos\ContentRepository\Core\SharedModel\ContentRepository\ContentRepositoryId;
+
+/**
+ * @internal
+ */
+final readonly class CommandHooksFactoryDependencies
+{
+    private function __construct(
+        public ContentRepositoryId $contentRepositoryId,
```

Comment:
As [discussed](https://neos-project.slack.com/archives/C04PYL8H3/p1730993828983519) this should contain more useful $stuff like access to projection states (and especially the content graph projection)

Reply:
II added the content graph read model with a3aa2266bb5d77fa55b64e6a76ca09aeb18c6e14